com.github.Qv2ray.json

```diff
@@ -0,0 +1,87 @@
+{
+    "app-id": "com.github.Qv2ray",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "qv2ray",
+    "rename-desktop-file": "qv2ray.desktop",
+    "rename-icon": "qv2ray",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--socket=session-bus",
+        "--filesystem=host",
+        "--device=dri",
+        "--filesystem=xdg-run/dconf",
```

Comment:
Dconf access isn't needed anymore for KDE 5.13/ FreeDesktop 19.08 / GNOME 3.34

Reply:
But I need it for gsettings.

Reply:
Have a better solution?

Reply:
It should work by default.

Reply:
Not for me. I am currently using Gnome 3.28 on Ubuntu 18.04.

Reply:
That depends on what you need gsettings for?

Reply:
org.gnome.system.proxy

Reply:
We need that to set the system proxy automatically.